app/src/main/java/acr/browser/lightning/settings/preferences/UserPreferencesExtensions.kt

```diff
@@ -13,20 +13,23 @@ fun UserPreferences.userAgent(application: Application): String =
         // WebSettings default identifies us as WebView and as WebView Google is preventing us to login to its services.
         // Clearly we don't want that so we just modify default user agent by removing the WebView specific parts.
         // That should make us look like Chrome, which we are really.
-         1 -> {
+        1 -> {
```

Comment:
I whish we could get ride of hard coded magic number. That would enable us to reorder the options as we see fit and make the code easier to maintain. For instance we could have the hide device UA come second in the list just after default I guess and custom come last.

Reply:
I agree. I also wanted to put `hide device` in second place, but the current design doesn't allow that.

Reply:
Feel free to step in. I don't mind if it means users user agent will be reset to default.